interpreters/src/insert.rs

```diff
@@ -174,13 +198,15 @@ impl<'a> TsidBuilder<'a> {
     }
 }
 
+/// Fill missing columns which can be calculated via default value expr.
 fn fill_default_values(
-    rows: &mut RowGroup,
-    default_value_map: &HashMap<usize, DfLogicalExpr>,
+    table: TableRef,
+    row_groups: &mut RowGroup,
+    default_value_map: &BTreeMap<usize, DfLogicalExpr>,
 ) -> Result<()> {
-    let input_df_schema = DFSchema::empty();
-    let input_arrow_schema = Arc::new(ArrowSchema::empty());
-    let input_batch = RecordBatch::new_empty(input_arrow_schema.clone());
+    let mut cached_columns_map: HashMap<usize, DfColumnarValue> = HashMap::new();
```

Comment:
It is not very elegant to suffix the variable with the data type, and how about such a name: `indexed_column_values`.

Reply:
`cached_column_values` is ok to me.